input/pagecontent/reference.md

```diff
@@ -176,13 +176,11 @@ FSH strings support the escape sequences that FHIR already defines as valid in i
 
 FHIR elements can contain [references to other Resources](https://www.hl7.org/fhir/R4/references.html#2.3.0). FSH represents references using the syntax `Reference({ResourceOrProfile})`. A name, id, or URL can be used to identify the resource or profile. For example, `Reference(USCorePatientProfile)`, `Reference(us-core-patient)`, and `Reference(http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient)` all are valid references to the [US Core Patient profile](http://hl7.org/fhir/us/core/structuredefinition-us-core-patient.html). When referring to a Reference element, the `Reference()` MUST be included, except in the case of a [reference choice path](#reference-paths). When syntax allows for multiple References, the items are separated by `or` placed *inside* the parentheses, e.g. `Reference(Patient or Practitioner)`, **not** `Reference(Patient) or Reference(Practitioner)`. 
 
-In constructing profiles, references typically refer to resource or profile *types*, for example, the subject of an Observation could be constrained to `Reference(Patient or Group)`. For instances, references typically refer to other instances, for example, a subject of an Observation could be `Reference(JaneDoe)`, assuming JaneDoe names a Patient instance.
-
-If there are no restrictions on the type of Resource or Profile an element can refer to, this is expressed as `Reference(Any)`.
```

Comment:
This is actually not true.  In a profile/extension it doesn't make any sense to _constrain_ a reference to `Reference(Any)` -- so it's irrelevant there.  In a logical model or resource, you simply specify the type as `Reference` (no parens at all) to indicate a reference to anything.

Reply:
To clarify, by "this is actually not true", what I mean is that we don't support the construct `Reference(Any)`.  Sushi complains that it can't find a resource called `Any`.